frontend/lib/norent/letter-builder/confirmation.tsx

```diff
@@ -7,18 +7,41 @@ import {
   USStateChoice,
 } from "../../../../common-data/us-state-choices";
 import { LetterBuilderAccordion } from "./welcome";
+import { getNorentMetadataForUSState } from "./national-metadata";
 
 const checkCircleSvg = require("../../svg/check-circle-solid.svg") as JSX.Element;
 
-const NORENT_FEEDBACK_FORM_URL = "https://airtable.com/shrrnQD3kXUQv1xm3";
+const NATIONAL_LEGAL_AID_URL = "https://www.lawhelp.org";
 const CANCEL_RENT_PETITION_URL = "https://cancelrent.us/";
+const NORENT_FEEDBACK_FORM_URL = "https://airtable.com/shrrnQD3kXUQv1xm3";
 
 export const NorentConfirmation: React.FC<{}> = () => {
   const { session } = useContext(AppContext);
   const letter = session.norentLatestLetter;
   const user = session.norentScaffolding;
-  const stateName =
-    user?.state && getUSStateChoiceLabels()[user?.state as USStateChoice];
+  const state = user?.state && (user?.state as USStateChoice);
```

Comment:
ohh you actually don't want to use the `state` value from `norentScaffolding` this far into the process: the `norentScaffolding`'s `state` value is actually only used to _temporarily_ store the value of the user's state while they are in the process of creating an account, because we don't yet have an onboarding info model in the DB to store it in.  That means that this code will work for users who went from not having an account all the way through to confirmation in one fell swoop, but it won't work for users who saved their progress for later, or for legacy tenant app users.

So to account for all cases, instead of pulling from `norentScaffolding` you will want to pull from `session.onboardingInfo.state`.  Sorry that's not clear!

Reply:
Ah ok, thanks for clarifying! I just added this quick change for now